canonical/ubuntu.com
# Fixed the search styling

## Done
- Re-factored the local search styling
- Went out on a limb for the mobile styling as it doesn't look great on live
## QA
- Check the search on large screen matches the search on live
- Reduce the screen width and see the search does not wraps but snaps to mobile just before
- Hover on the search on small screens and see you can reach the search and see the styles
## Issue / Card

Fixes https://github.com/ubuntudesign/www.ubuntu.com/issues/188
